src/languageSupport/languages/flux/parser/index.ts

```diff
@@ -27,10 +27,10 @@ export const parse = (script): File => {
   }
 }
 
-export const format_from_js_file = (script): string => {
+export const format_from_js_file = (script: File): string => {
   if (window) {
     return flux_format_from_js_file(script)
   } else {
-    return script
+    return ''
```

Comment:
Check with @genehynson on this change.

Reply:
I'm unsure this has ever been used properly. The change happened because adding type information makes in clear that we were _saying_ we'd return a string, but in this case, we're returning a AST object. I don't think this ever _actually_ affected anything, the tests pass, and it's close to the raw AST we're already returning.

Reply:
I did some digging on how this function is used and I think this change is fine. It's probably safer anyways to return `''` here instead of the AST obj that we provide as `script` ✅